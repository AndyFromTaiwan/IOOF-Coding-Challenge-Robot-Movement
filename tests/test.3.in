PLACE 0,0,EAST
REPORT
MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
LEFT
REPORT

MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
LEFT
REPORT

MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
LEFT
REPORT

MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
LEFT
REPORT

PLACE 4,4,SOUTH
REPORT
MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
RIGHT
REPORT

MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
RIGHT
REPORT

MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
RIGHT
REPORT

MOVE
MOVE
MOVE
MOVE
REPORT
MOVE
REPORT
RIGHT
REPORT