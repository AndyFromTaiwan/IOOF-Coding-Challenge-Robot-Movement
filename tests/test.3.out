0,0,EAST
4,0,EAST
4,0,EAST
4,0,NORTH
4,4,NORTH
4,4,NORTH
4,4,WEST
0,4,WEST
0,4,WEST
0,4,SOUTH
0,0,SOUTH
0,0,SOUTH
0,0,EAST
4,4,SOUTH
4,0,SOUTH
4,0,SOUTH
4,0,WEST
0,0,WEST
0,0,WEST
0,0,NORTH
0,4,NORTH
0,4,NORTH
0,4,EAST
4,4,EAST
4,4,EAST
4,4,SOUTH